src/EFCore.PG/Diagnostics/NpgsqlEventId.cs

```diff
@@ -57,107 +58,98 @@ private enum Id
             UnsupportedConstraintIndexSkippedWarning
         }
 
-        private static readonly string _validationPrefix = DbLoggerCategory.Model.Validation.Name + ".";
-        private static EventId MakeValidationId(Id id) => new EventId((int)id, _validationPrefix + id);
-
-        /// <summary>
-        ///     <para>
-        ///         No explicit type for a decimal column.
-        ///     </para>
-        ///     <para>
-        ///         This event is in the <see cref="DbLoggerCategory.Model.Validation" /> category.
-        ///     </para>
-        ///     <para>
-        ///         This event uses the <see cref="PropertyEventData" /> payload when used with a <see cref="DiagnosticSource" />.
-        ///     </para>
-        /// </summary>
-        //public static readonly EventId DecimalTypeDefaultWarning = MakeValidationId(Id.DecimalTypeDefaultWarning);
-
-        /// <summary>
-        ///     <para>
-        ///         A byte property is set up to use a SQL Server identity column.
-        ///     </para>
-        ///     <para>
-        ///         This event is in the <see cref="DbLoggerCategory.Model.Validation" /> category.
-        ///     </para>
-        ///     <para>
-        ///         This event uses the <see cref="PropertyEventData" /> payload when used with a <see cref="DiagnosticSource" />.
-        ///     </para>
-        /// </summary>
-        //public static readonly EventId ByteIdentityColumnWarning = MakeValidationId(Id.ByteIdentityColumnWarning);
-
-        private static readonly string _scaffoldingPrefix = DbLoggerCategory.Scaffolding.Name + ".";
-        private static EventId MakeScaffoldingId(Id id) => new EventId((int)id, _scaffoldingPrefix + id);
-
-        /// <summary>
-        ///     A column was found.
-        ///     This event is in the <see cref="DbLoggerCategory.Scaffolding" /> category.
+// TODO: Is there any reason to keep this?
```

Comment:
I guess not, they were meant to remind me that the event porting work from MSSQL wasn't done, but these events really don't seem relevant to Npgsql.